doc/examples/plugins-hmac-auth-generate-signature.md

```diff
@@ -0,0 +1,184 @@
+
```

Comment:
Need to add LICENSE header: https://github.com/apache/apisix/runs/1345989159#step:11:136 to make the CI pass.

Reply:
Thanks, I will modify the PR